testing_update

```diff
@@ -0,0 +1,5 @@
+Testing Update as of 1/19/2019
+
+Some people got recommendations but for those who weren't located in large cities, the API call resulted in a key value error. This was combatted by integrating the ProPublica API (allowing for unlimited calls using zip codes). The key value error was due to towns or a call resulting in only a zip code and not an actual city. For this reason, there was an error in giving back a suggestion. If trips are not cleaned, no suggestion will appear until the trip has been cleaned and a mode of transportation is associated with it. The suggestion system has been updated to guesstimate endpoints, since in some areas it can be difficult to pinpoint a restaurant (malls, parking lots, etc). 
```

Comment:
this is pretty close to what I was expecting.
Maybe with steps like:
- trip to coffee shop in Milpitas; suggestion generated to ....
- trip to friend's home in OC suburb; suggestion generated although it shouldn't be
....

Reply:
also, have you tested this end to end? If the testing is happening in an ipython notebook, then how is the UI talking to the changed server?

Reply:
Yes sorry. To clarify, end-to-end its displaying properly unless there is an error on the backend, in which case the screen simply displays "loading" continuously. If that is the case, we specifically have been debugging those individual trips in the ipython notebook. Once the debugging is completed, the trips seem to show up normally. I added some sample tests and suggestions.